app/src/main/java/ch/hikemate/app/utils/DatetimeUtils.kt

```diff
@@ -0,0 +1,85 @@
+package ch.hikemate.app.utils
+
+import android.content.Context
+import ch.hikemate.app.R
+import com.google.firebase.Timestamp
+import java.time.Duration
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+import java.time.format.TextStyle
+import java.util.Locale
+
+/**
+ * Create a [Timestamp] from a year, month, and day.
+ *
+ * For example, from(2022, 1, 1) will create a [Timestamp] for January 1st, 2022.
+ *
+ * @param year No offset needed (for a date in 2024, use 2024). Needs to be between 1 and 9999 (both
+ *   inclusive).
+ * @param month Month of the desired timestamp. 1-based (January is 1, February is 2, ...). Needs to
+ *   be between 1 and 12 (both included).
+ * @param day Day of the desired timestamp. 1-based. Must be within the range of the given month.
+ *   Providing an invalid day might result in an unexpected timestamp.
+ * @return A [Timestamp] representing the given year, month, and day.
+ */
+fun Timestamp.Companion.from(year: Int, month: Int, day: Int): Timestamp {
```

Comment:
Good catch ! I wrote some additional tests for this function to check that invalid values throw an exception. Exceptions are already being thrown by the underlying libraries that are used (`LocalDateTime` and `Timestamp`), so I simply added tests to check that behavior to avoid it changing in the future.

Added in [095e493a](https://github.com/HikeMate/hikeMateApp/pull/77/commits/095e493a93b1e17b60a4996daca2344cd3e881f2).